bridge_adaptivity/module/templates/module/collection_detail.html

```diff
@@ -106,115 +107,19 @@ <h4 class="panel-title">
-                          .text(" ")
-                          .css('margin-right', '5px')
-                          .appendTo(listItem);
-                      var sourceButton = $('<span/>')
-                          .attr('data-toggle', 'modal')
-                          .attr('data-target', '#activityModal')
-                          .text(item['display_name'])
-                          .appendTo(listItem);
-                      sourceButton.on('click', function () {
-                          setInitialActivityData(item);
-                      });
-                      createPreviewButton(
-                          item['display_name'],
-                          item['lti_url'],
-                          item['id'],
-                          listItem,
-                          modalContentFrame
-                      )
-                  });
-                  content_panel.append(sourcesList);
-              });
-          });
-
-          function createPreviewButton(title, ltiUrl, sourceId, parent, modalFrame) {
-              var preview = $('<a/>')
-                  .addClass("pull-right")
-                  .appendTo(parent);
-              var previewButton = $('<button/>', {'class': 'btn btn-default btn-sm'})
-                  .appendTo(preview);
-              var previewButtonBody = $('<span/>')
-                  .addClass("glyphicon glyphicon-eye-open")
-                  .attr('data-toggle', 'modal')
-                  .attr('data-target', '#sourceModal')
-                  .attr('data-display-name', title)
-                  .attr('data-lti-url', ltiUrl)
-                  .appendTo(previewButton);
-              previewButton.on('click', function () {
-                  $('#sourceModalLabel').text(title);
-                  configurePreview(modalFrame, title, ltiUrl, sourceId);
-              })
-          }
-
-          function configurePreview(frame, title, ltiUrl, sourceId) {
-              var idParam = "source_id=" + sourceId + "&";
-              var displayNameParam = "source_name=" + title + "&";
-              var ltiUrlParam = "source_lti_url=" + ltiUrl + "&";
-              var previewUrl = "{% url 'lti:source-preview' %}?" + idParam + displayNameParam + ltiUrlParam;
-              frame
-                  .attr('src', previewUrl)
-                  .attr('title', title)
-                  .attr('name', sourceId);
-          }
-
-          function setInitialActivityData(source) {
-              $('#id_name').val(source['display_name']);
-              $('#id_source_name').val(source['display_name']);
-              $('#id_source_launch_url').val(source['lti_url']);
-              $('#id_source_context_id').val(source['context_id']);
-          }
-      }(jQuery));
+  <script type="application/javascript">
+      var activitiesData = {{activities_data | safe}};
+      var internalUrls = {
+          apiSources: "{% url 'api:sources' %}",
```

Comment:
Please use `data-` attributes to send variables to the JS

Reply:
Sorry for misleading I meant data attribute in the html file inside appropriate tag instead of adding variables into the global namespace.